docs/security.rst

```diff
@@ -56,10 +56,18 @@ semantics as for ``x-tokenInfoFunc``, but the function accepts three
 parameters: username, password and required_scopes. If the security declaration
 of the operation also has an oauth security requirement, required_scopes is
 taken from there, otherwise it's None. This allows authorizing individual
-operations with oauth scope while using basic authentication for
+operations with `oauth scope`_ while using basic authentication for
 authentication.
 
-You can find a `minimal Basic Auth example application`_ in Connexion's "examples" folder.
+You can find a `minimal Basic Auth example application`_ in Connexion's "examples" folder:
```

Comment:
maybe we could also point to the readme of the example

Reply:
Yes, you're right. I removed the inline code, and used the link instead.